app/main.py

```diff
@@ -1,3 +1,8 @@
+from typing import Dict, Any
+from app.knights import Knight
+from app.battle import Battle
+
+
 KNIGHTS = {
```

Comment:
please move dict with all data to separate module

Reply:
done